### ISSUE TITLE
Accessibility:Step Input: The error message “Value is required” is not read when step input is accessed by the screen reader

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3490]
Description: The error message “Value is required” is not read when step input is accessed by the screen reader

Expected: Error message should be conveyed by the screen reader when UI is accessed

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94649907-46156000-0313-11eb-8b07-7fb0aee5bf49.png)

**Comment by puru-hk** [comment]
Tested it is working as expected.
